_data/meltano/utilities/dagster/quantile-development.yml

```
@@ -0,0 +1,51 @@
+name: dagster
+label: Dagster
+domain_url: https://dagster.io/
+namespace: dagster
+pip_url: dagster-ext
+ext_repo: https://github.com/quantile-development/dagster-ext
+maintenance_status: development
+repo: https://github.com/dagster-io/dagster
+executable: dagster_invoker
+logo_url: /assets/logos/utilities/dagster.png
+variant: quantile-development
+commands:
+  describe:
+    executable: dagster_extension
+    args: describe
+    description: Describe the Dagster Extension.
+  init:
+    args: initialize
+    executable: dagster_extension
+    description: Initialize the Dagster extension.
+  up:
+    args: -f $REPOSITORY_DIR/repository.py
+    executable: dagit_invoker
+    description: Start Dagster.
+settings:
+- name: dagster_home
+  label: Dagster Home
+  env: DAGSTER_HOME
+  value: $MELTANO_PROJECT_ROOT/.meltano
+  description: The home folder of Dagster.
+- name: repository_dir
+  label: Repository Directory
+  env: REPOSITORY_DIR
+  value: $MELTANO_PROJECT_ROOT/orchestrate
+  description: The root folder of the Dagster project.
+keywords:
+- dagster
+- meltano_edk
+definition: is an [orchestrator](https://docs.meltano.com/concepts/plugins#orchestrators) that allows for workflows to be programmatically authored, scheduled, and monitored.
+next_steps: |
+  ### Initialize your Dagster project
+
+  ```bash
+  meltano invoke dagster:initialize
```

```suggestion
  meltano invoke dagster:init
```

@JulesHuisman this is awesome! I got it setup so fast. I got this error at first though `Command 'initialize' could not be found. Utility 'dagster' supports the following commands: describe, init, up` , I think the docs were just out of sync with the commands.

---

@pnadolny13 Glad you like it! 

Ah I see now @tayloramurphy removed the `initialize` command. I had both `initialize` and `init` as commands, as `initialize` is the default for other edk extensions, while `init` is usually what is used by other cli tools (`poetry init`, `meltano init`).

I am not sure if you guys have any opinion on this, to just have one command (and keep it steady across all edk extensions) or if it is acceptable to have `init` as an alias.

---

Let's keep it as initialize - I made a suggestion on the command itself to change it back.